File: src/main/org/nlogo/prim/etc/_stamp.java
package org.nlogo.prim.etc;

import org.nlogo.nvm.Command;
import org.nlogo.nvm.Syntax;

public final strictfp class _stamp
    extends Command {
  @Override
  public Syntax syntax() {
    return Syntax.commandSyntax("-T-L", true);
  }

  @Override
  public void perform(final org.nlogo.nvm.Context context) {
    world.stamp(context.agent, false);
    context.ip = next;
  }
}

File: src/main/org/nlogo/prim/etc/_stamp.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.prim.etc

import org.nlogo.api.Syntax
import org.nlogo.nvm.{ Command, Context }

class _stamp extends Command {
  override def syntax =
    Syntax.commandSyntax("-T-L", true)
  override def perform(context: Context) {
    world.stamp(context.agent, false)
    context.ip = next
  }
}
